Makefile

```diff
@@ -59,6 +60,9 @@ COMPONENTS_USAGE_STATS_FILES = $(shell \
 		-type f \
 		\( -name '*.js' -or -name '*.jsx' \) \
 )
+COMPONENTS_PROPTYPE_FILES = $(shell \
+	find client -name 'index.jsx' -or -name 'example.jsx' \
```

Comment:
We sometimes use the `.js` extension, even if the component contains JSX. e.g. `components/signup-site-title/index.js`, we should probably account for these, naming philosophy aside. We'll probably want to ignore `test/` files as well.

Also, what do you think about limiting the files we scan to `blocks` and `components`?